### PR TITLE
Profile header component

### LINKED ISSUE #19: Create Profile Header Component
Create Profile Header Component with the following elements:

- Avatar
- Username
- Likes count with heart icon
- Comment count with a comment icon

### PR DESCRIPTION
**Pull request by MihaelaCatan04**

### CODE REVIEW

File: src/components/ProfileHeader/index.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import ProfileHeader from "./ProfileHeader";
+export default ProfileHeader;
```
Comment (vbetivu-ellation) [inline src/components/ProfileHeader/index.js]: Let's add an empty line between the statements and one at the end of the file
```suggestion
import ProfileHeader from "./ProfileHeader";

export default ProfileHeader;

```

File: src/components/ProfileHeader/ProfileHeader.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { FilledLikeSvg, Avatar, CommentSvg, Text } from "../common";
+import styles from "./ProfileHeader.module.css";
+import PropTypes from "prop-types";
+
+const ProfileHeader = ({username, avatar, likescount, commentscount}) => (
```
Comment (vbetivu-ellation) [inline src/components/ProfileHeader/ProfileHeader.js]: Let's use camelCase
```suggestion
const ProfileHeader = ({username, avatar, likesCount, commentsCount}) => (
```

File: src/components/ProfileHeader/ProfileHeader.module.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+.Avatar {
+  max-width: 200px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 25px;
+}
+
+.Text {
+  text-align: center;
```
Comment (vbetivu-ellation) [inline src/components/ProfileHeader/ProfileHeader.module.css]: text-align is already declared on parent (ProfileHeader)
```suggestion
```

File: src/components/ProfileHeader/ProfileHeader.module.css [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+.Avatar {
+  max-width: 200px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 25px;
+}
+
+.Text {
```
Comment (vbetivu-ellation) [inline src/components/ProfileHeader/ProfileHeader.module.css:19]: Let's add some bottom margin, to have more space between items

File: src/components/ProfileHeader/ProfileHeader.module.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  text-align: center;
+  align-self: stretch;
```
Comment (vbetivu-ellation) [inline src/components/ProfileHeader/ProfileHeader.module.css]: This prop does not have any effect
```suggestion
```

File: src/components/ProfileHeader/ProfileHeader.module.css [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+  align-items: center;
+}
+
+.FilledLike {
+  text-align: center;
+}
+
+.Comment {
+  text-align: center;
+}
```
Comment (vbetivu-ellation) [inline src/components/ProfileHeader/ProfileHeader.module.css]: text-align is already declared on parent (ProfileHeader)

File: src/components/ProfileHeader/ProfileHeader.module.css [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+  fill: tomato;
+}
+
+@media (min-width: 568px) {
+  .ProfileHeader {
+    flex-direction: row;
+  }
+  .Avatar {
+    margin-left: 0%;
+    margin-right: 0%;
```
Comment (vbetivu-ellation) [inline src/components/ProfileHeader/ProfileHeader.module.css]: Let's add some space between the image and info
```suggestion
    margin: 0 32px 0 0;
```

File: src/components/ProfileHeader/ProfileHeader.module.css [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+  }
+  .Text {
+    text-align: unset;
+    align-self: unset;
+    margin-right: 8px;
+  }
+  .Center {
+    flex-direction: row;
```
Comment (vbetivu-ellation) [inline src/components/ProfileHeader/ProfileHeader.module.css]: let's leave the flex-direction as column, but align the items to the start

File: src/components/ProfileHeader/ProfileHeader.module.css [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+.Center {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.FilledLike {
+  text-align: center;
+}
```
Comment (vbetivu-ellation) [inline src/components/ProfileHeader/ProfileHeader.module.css]: text-align is already declared on parent (ProfileHeader)

File: src/components/ProfileHeader/ProfileHeader.module.css [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+  .Text {
+    text-align: unset;
+    align-self: unset;
+    margin-right: 8px;
+  }
+  .Center {
+    flex-direction: row;
+  }
+  .FilledLike {
+    margin-right: 10px;
```
Comment (vbetivu-ellation) [inline src/components/ProfileHeader/ProfileHeader.module.css]:
```suggestion
    margin-bottom: 12px;
```

File: src/components/ProfileHeader/ProfileHeader.module.css [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+  align-self: stretch;
+}
+
+.Center {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.FilledLike {
```
Comment (vbetivu-ellation) [inline src/components/ProfileHeader/ProfileHeader.module.css:29]: let's make it flex and center the items
```suggestion
.FilledLike {
  display: flex;
  align-items: center;
```

File: src/components/ProfileHeader/ProfileHeader.module.css [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+  text-align: center;
+}
+
+.Comment {
```
Comment (vbetivu-ellation) [inline src/components/ProfileHeader/ProfileHeader.module.css:34]: let's make it flex and center the items
```suggestion
.FilledLike {
  display: flex;
  align-items: center;
```

**Review by ileahu-ellation — APPROVED** [review]

**Pull request merged**